src/System.Management/tests/System/Management/ManagementClassTestsMofRequired.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Management.Tests
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "WMI not supported via UAP")]
+    [Collection("Mof Collection")]
+    public class ManagementClassTestsMofRequired
+    {
+        [ConditionalFact(typeof(WmiTestHelper), nameof(WmiTestHelper.IsElevatedAndNotNanoServer))]
+        public void Create_Modify_Delete_Static_Class()
+        {
+            using (var newClass = new ManagementClass(WmiTestHelper.Namespace))
+            {
+                const string NewClassName = "CoreFX_Create_Modify_Delete_Static_Class森鷗外";
```

Comment:
Please use Unicode escape sequences as they don't rely on tools correctly interpreting and preserving file encoding.

In this case `\uEE68\uD79D\u1659`

Reply:
Which seems to be some Japanese author. But maybe you knew that.